draft-rosa-bmwg-vnfbench

```diff
@@ -263,12 +273,20 @@ Table of Contents
       can define parameter studies.
 
    In general, automated VNF benchmarking Tests must capture relevant
-   causes of performance variability.  To dissect a benchmarking Test,
-   different stages of benchmarking procedures are defined by generic
-   operations that may or may not be automated, categorized in the
-   sections that follow.  In each one of them, influencing aspects on
-   the performance of a VNF must be carefully analyzed while automating
-   a VNF benchmarking methodology, as explained in sequence.
+   causes of performance variability.  To dissect a VNF benchmarking
+
+
+
+Rosa, et al.            Expires November 22, 2019               [Page 5]
+
+Internet-Draft                  VNFBench                        May 2019
+
+
+   Test, in the sections that follow different benchmarking phases are
+   categorized defining generic operations that may be automated.  When
+   automating a VNF benchmarking methodology, in each one of the phases
+   explained in sequence, all the influencing aspects on the performance
+   of a VNF must be carefully analyzed and comprehensively reported.
```

Comment:
Idea for reformulation: When automating a VNF benchmarking methodology, all the influencing aspects on the performance
   of a VNF must be carefully analyzed and comprehensively reported, in each phase of the overall benchmarking process.

Reply:
Applied